docs/create_new_activity_protocol.md

```diff
@@ -0,0 +1,131 @@
+but does cover most cases. A template of the CSV and how to use the tool can be found
+[here](https://github.com/sanuann/reproschema-builder)
+
+## Manual schema generation
+
+Fork the project and manually create the jsonld files according to the above directory structure.
+(This process will be tedious for large questionnaires).
+
+### Create an activity
+
+Under the [`activities`](./activities) directory, create directory with name of activity in the
+`CamelCase` naming convention.
+
+Activity directory structure:
+
+-   `/items` (directory) : contains the `jsonld` files for individual items of the activity schema
+    -   `item_1`
+    -   `item_2`
+    -   …
+-   `activityName_schema` : schema to define the activity
+-   `activityName_context` : context to define keys used specific to the activity schema
+
+Creating `activityName_schema` – use the keys defined in [`schemas/Activity`](./schemas/Activity).
+If any other keys are used, then define them in `activityName_context`
+
+For example,
+
+```json
+  {
+  "@context": [ "https://raw.githubusercontent.com/ReproNim/reproschema/master/contexts/generic",
+      "https://raw.githubusercontent.com/ReproNim/reproschema/master/activities/Wellbeing/Wellbeing_context"
+  ],
+  "@type": "reproschema:Activity",
+  "@id": "Wellbeing_schema",
+  "skos:prefLabel": "Wellbeing",
+  "schema:description": " Wellbeing Voice tasks",
+  "schema:schemaVersion": "0.0.1",
+  "schema:version": "0.0.1",
+  "preamble": {
+      "en": "For each task, you should hit the record button before speaking and then stop once you are done speaking. You may hit play to hear what was recorded.",
+      "es": "Para cada pregunta, debería hacer click sobre el botón antes de hablar y luego hacerlo de nuevo para parar de grabar. Luego puede tocar play para escuchar su respuesta."
+  },
+  "ui": {
+      "addProperties": [
+          {"isAbout": "share_data",
+          "variableName": "share_data",
+          "prefLabel": {"en": "Share Data"},
+          "isVis": true,
+          "allow": ["skipped", "dontknow"],
+          ...
+          },
+          {"isAbout": "email",
+          "variableName": "email",
+          "isVis": "share_data === 1"
+          },
+          {"isAbout": "multipart_audio_check",
+          "variableName": "multipart_audio_check",
+          "isVis": true
+          },
+          {"isAbout": "free_speech_general_mood",
+          "variableName": "share_data",
+          "isVis": true
+          },
+          {"isAbout": "say_ah",
+          "variableName": "say_ah",
+          "isVis": true
+          }
+      ],
+      "order": [
+          "multipart_audio_check",
+          "share_data",
+          "email",
+          "free_speech_general_mood",
+          "say_ah"
+      ],
+      "shuffle": false
+  }
+}
+```
+
+Mandatory keys:
+
+-  `@context` - [Array] Include the ReproNim generic context JSON-LD file along with the activity context.
```

Comment:
I am a big fan of tables so I really like that you created a separate doc with the schema properties. :heart: 

How much work would it be to actually add a column those tables that indicates which property is mandatory? And we put those properties at the top of the table.

I think you created that property doc with a parser so I realize that what I am asking can be a pain. :see_no_evil:  

Let know what you think?

Also

Reply:
the parser basically reads the schema from `schemas` directory and creates the properties table. the "required field" info isn't present in the schemas. So do you think I should add that manually to each of the table?

Also, I had seen you put in some extra headers/text in the properties doc. (I lost track where and when I saw it in your commits). But every time the schema is updated, this script should run and generate the properties doc. So those extra text which you put in the doc would vanish. no? perhaps it should be put somewhere in the script?
For reference I linked the script I used to generate the properties markdown: https://github.com/sanuann/reproschema/blob/documentation/validation/schema2markdown.py

Reply:
OK let's not add that manually. 

Maybe we should have the required properties listed somewhere so that this extra column can be generated by your script.

Yes if your script could add the header, it would be fab because the rendering of the doc will be easier on the eye of the reader in the end.

Reply:
Actually I was thinking that it might be easier if we get a different markdown file for the properties of each schema, what do you think?

Reply:
actually, initially I did it that way - separate for each file but then thought it was unnecessary. I can always split it. Is there any advantage?